app/javascript/packs/state-guidance.ts

```diff
@@ -10,11 +12,27 @@ export function showOrHidePuertoRicoExtras(forStateCode) {
   });
 }
 
+export function showOrHideJurisdictionExtras(forJurisdictionCode) {
+  document.querySelectorAll('.jurisdiction-extras').forEach((element) => {
+    switch (forJurisdictionCode) {
+      case 'TX':
+        element.innerHTML = t('in_person_proofing.form.state_id.state_id_number_texas_hint');
+        break;
+      case 'FL':
+        element.innerHTML = t('in_person_proofing.form.state_id.state_id_number_hint_html');
+        break;
```

Comment:
are we going to add FL-specific text?
if not, we can just let this call through to the default, right?
```suggestion
```

Reply:
I added this as a placeholder as we will add florida specific text in another ticket. But I can remove it for this pr.

Reply:
Yeah it's very easy to add to later, I'd agree and remove from this PR